### PR TITLE
Replace individual cli tests with parametrized tests

### PR DESCRIPTION
**Pull request by achals**
Signed-off-by: Achal Shah <achals@gmail.com>

**What this PR does / why we need it**:

More coverage with fewer test files.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: sdk/python/tests/integration/registration/test_cli.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from tests.integration.feature_repos.universal.data_source_creator import (
+    DataSourceCreator,
+)
+from tests.utils.cli_utils import CliRunner, get_example_repo
+from tests.utils.online_read_write_test import basic_rw_test
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize("test_repo_config", FULL_REPO_CONFIGS)
+def test_universal_cli(test_repo_config) -> None:
```
Comment (adchia) [inline sdk/python/tests/integration/registration/test_cli.py:20]: test_cli_local seems to have a lot of tests that check other commands. Do we want to add them here too?

Comment (achals) [reply sdk/python/tests/integration/registration/test_cli.py:20]: Yep good point

Comment (achals) [reply sdk/python/tests/integration/registration/test_cli.py:22]: Added them back.

**Review by adchia — APPROVED** [review]
/lgtm

**Pull request merged**